packages/app/src/vis-packs/core/hooks.ts

```diff
@@ -28,7 +28,15 @@ export const useToNumArray = createMemo(toNumArray);
 
 export function useToNumArrays(
   arrays: ArrayValue<NumericLikeType>[],
-): NumArray[] {
+): NumArray[];
+
+export function useToNumArrays(
+  arrays: (ArrayValue<NumericLikeType> | undefined)[],
```

Comment:
In mapped components, the `auxErrors` and `axisValues` array props are allowed to have `undefined` values.